minecraft_servers/tornadomc/manifest.json

```diff
@@ -9,10 +9,10 @@
     "de"
   ],
   "social": {
-    "web": "https://forum.tornadomc.net",
+    "web": "https://tornadomc.net",
```

Comment:
Does not load
https://i.imgur.com/b1WrTi2.png

Reply:
Fixed

Reply:
Still not loading